docs/content/developers/release-management.md

```diff
@@ -61,38 +61,19 @@ The following “Repository secret” environment variables must be added to <ht
 ### Post-Release Tasks
 
 1. After the release has been created, the new gitpod image must be pushed.
-   1. `cd .gitpod/images && DOCKER_TAG="<YYMMDD>" ./push.sh
-   2. PR to update .gitpod.yml with the new image.
+   1. `cd .gitpod/images && DOCKER_TAG="<YYMMDD>" ./push.sh`
+   2. PR to update `.gitpod.yml` with the new image.
    3. PR to update [ddev-gitpod-launcher](https://github.com/ddev/ddev-gitpod-launcher) with the new image.
 
 ## Pushing Docker Images with the GitHub Actions Workflow
 
 The easiest way to push Docker images is to use the GitHub Actions workflow, especially if the code for the image is already in the [ddev/ddev](https://github.com/ddev/ddev) repository.
 
-### Prerelease tasks
-
-1. Push the new version of drud/ddev-php-base
-2. Update drud/ddev-webserver to use the new version of drud/ddev-php-base and push it with the proper tag
-3. Make sure the `version-history.md` file is up to date.
-4. Make sure the docker images for the new tag are all tagged and pushed.
-5. Make sure the pkg/version/version.go is all set to point to the new images (and tests have been run)
-6. Make sure you're about to create the right release tag.
-
-### Actual release creation
-
-1. Create a release for the new version using the GitHub UI. It should be "prerelease" if it's an edge release. Use the right tag!
-2. Use the "Auto-generate release notes" option to get the commit list, then edit to add all the other necessary info.
-3. Verify that homebrew (linux and macOS) and Chocolatey and AUR are working correctly with the right versions
-
-## Pushing docker images with the GitHub Actions workflow
-
-The easiest way to push docker images is to use the GitHub Actions workflow, especially if the code for the image is already in the ddev repo.
-
```

Comment:
I removed the entire section because I suspect a copy and paste from the section above.

Reply:
Sorry, this isn't correct. The two sections need to be merged, not just deleted. The other one doesn't have the pushing of ddev-php-base, for example.

Reply:
Done, I just removed the duplicated text:
```
## Pushing docker images with the GitHub Actions workflow
The easiest way to push docker images is to use the GitHub Actions workflow, especially if the code for the image is already in the ddev repo.
```